scripts/build_dev_server_docker

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o xtrace
+
+readonly CACHE_DIR='bazel-cache'
+readonly SCRIPTS_DIR="$(dirname "$0")"
+
+"$SCRIPTS_DIR/docker_run" bazel \
```

Comment:
AFAICT there is no reason why this script needs to be tied to docker_run. I would rename it to `build_dev_server`, and then invoke it as an argument to `docker_run` in `.travis.yml`.

Reply:
Oh, good point. Done